lib/iiif_print/lineage_service_decorator.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+# OVERRIDE: IIIF Print v1.0.0 to work with slugs
+IiifPrint::LineageService.module_eval do
```

Comment:
Since this is a slug-related override, I was wondering if there is a reason it wasn't added to slug_override.rb (I haven't looked into it.. not everything works there, but most overrides do).

Reply:
slug_override.rb IS the full set of decorators for everything slug-related here. So if we do an override, I'd prefer it go in there.